python-lib/osisoft_plugin_common.py

```diff
@@ -278,7 +278,7 @@ def remove_unwanted_columns(row):
         row.pop(unwated_column, None)
 
 
-def format_output(input_row, reference_row=None, is_enumeration_value=False):
+def format_output(input_row, reference_row=None, is_enumeration_value=False, add_element_name=False):
```

Comment:
❓ Do we really need to have this flag here? We are always adding the element so no need to make it conditional AFAIK. What's the benefit of having the flag here?

Reply:
Indeed, it's not really used... Removed in [d516709](https://github.com/dataiku/dss-plugin-osisoft/pull/72/commits/d5167090f7507f23b661ed2c25fbbbb9c3f77ee9)